airflow/www/templates/admin/master.html

```diff
@@ -45,6 +45,13 @@
         }
       }
     });
+
+var el = document.createElement("span");
+
+export function escapeHtml(text) {
```

Comment:
```suggestion
function escapeHtml(text) {
```
export is supported only in external module.

Reply:
Whops yes, thanks.